### PR TITLE
Corrected ligo storage

### PR DESCRIPTION
**Pull request by claudebarde**

### CODE REVIEW

File: taqueria/.taq/development-state.json [hunk 1]
```diff
@@ -1,4 +1,135 @@
 {
     "operations": {},
```
Comment (michaelkernaghan) [inline taqueria/.taq/development-state.json]: Does this file need to include in the PR?

File: taqueria/.taq/state.json [hunk 2]
```diff
@@ -1,16 +1,24 @@
 // WARNING: This file is autogenerated and should NOT be modified
 {
```
Comment (michaelkernaghan) [inline taqueria/.taq/state.json]: Do we need this file in the merge a well?

**Review by michaelkernaghan — APPROVED** [review]
what is this .DSstore ?

File: taqueria/.taq/state.json (hunk 2 above)
Comment (hu3man) [reply taqueria/.taq/state.json]: Ya this should be in .gitignore

File: taqueria/.taq/development-state.json (hunk 1 above)
Comment (hu3man) [reply taqueria/.taq/development-state.json]: No, it's actually not even used in Taqueria currently..... state API... 😢

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 node_modules
-.env
\ No newline at end of file
+.env
+.DS_Store
```
Comment (ac10n) [inline .gitignore]: @claudebarde I suggest keeping an empty line at the end of all files. If we do it, adding new content only changes the added lines.
If we don't, we get an extra newline character at the end of last line.

This is a minor nuisance when doing code review, but a bit of more serious one when checking out the history of a line (or when your IDE shows you the last person and commit when a line is changed). They will all miss the real one.

Same reason for adding commas to last items in lists (when items are on separate lines) and other separators that are not necessary for the last item.

Comment (ac10n) [reply .gitignore]: Also, your `editorconfig` file can be set to add that automatically.

**Pull request merged**